src/Makefile.am

```diff
@@ -17,6 +17,8 @@ libvecdummy_la_SOURCES = testsuite/vec_int128_dummy.c \
 	testsuite/vec_bcd_dummy.c \
 	testsuite/vec_char_dummy.c
 
+libpvec_la_LIBADD = -lc
```

Comment:
This will cause the .la file to have a dependency on libc, as can be seen in the following diff between version 1.0.1 and HEAD:

```
diff -ru --color extract-1.0.1-1/usr/lib/powerpc64le-linux-gnu/libpvec.la extract-1.0.2-1/usr/lib/powerpc64le-linux-gnu/libpvec.la
--- extract-1.0.1-1/usr/lib/powerpc64le-linux-gnu/libpvec.la    2019-06-17 22:19:07.000000000 -0300
+++ extract-1.0.2-1/usr/lib/powerpc64le-linux-gnu/libpvec.la    2019-07-11 12:02:19.000000000 -0300
@@ -17,7 +17,7 @@
 inherited_linker_flags=''
 
 # Libraries that this one depends upon.
-dependency_libs=''
+dependency_libs=' -lc'
 
 # Names of additional weak libraries provided by this library
 weak_library_names=''
```

And this causes Debian's lintian to complain, with the following message:

```
E: libpvec-dev: non-empty-dependency_libs-in-la-file usr/lib/powerpc64le-linux-gnu/libpvec.la                                                                               
N:
N:    The dependency_libs field in the .la file has not been cleared. It has
N:    long been a release goal to get rid of unneeded .la files and clearing
N:    the dependency_libs field from the rest of them.
N:
N:    A non-empty dependency_libs field will also stall the Multi-Arch
N:    conversion.
N:
N:    The .la file in itself may be useful if the library is loaded
N:    dynamically via libltdl.
N:
N:    Refer to https://wiki.debian.org/ReleaseGoals/LAFileRemoval,
N:    https://lists.debian.org/debian-devel/2011/05/msg01003.html, and
N:    https://lists.debian.org/debian-devel/2011/05/msg01146.html for details.
N:
N:    Severity: serious, Certainty: certain
N:
N:    Check: shared-libs, Type: binary, udeb
```

I think I can fix this on Debian side, as this seems to be a Debian-specific decision to avoid .la dependencies.

I'll get back to you when I learn how to fix this (on the packaging... I don't think this needs fixing upstream).

Reply:
The fix for Debian is as trivial as removing this line, which I can certainly do with a downstream patch.

Reply:
Ok will commit this as is. Eventually libpvec will grow up to be a real library which will resolve this.